todo_app/templates/index.html

```diff
@@ -7,43 +7,47 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <div class="row">
+    <div class="col-md-12">
+      <h2>New item</h2>
+      <form action="{{ url_for('add_item') }}" method="POST">
+        <div class="form-group form-row align-items-center">
+          <div class="col-auto">
+            <label for="title-input">Title</label>
+          </div>
+          <div class="col">
+            <input type="text" name="title" id="title-input" class="form-control">
+          </div>
+          <div class="col-auto">
+            <button type="submit" class="btn btn-success">Add Item</button>
+          </div>
+        </div>
+      </form>
+    </div>
+  </div>
+
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-          <table> 
-            <th>ID</th>
-            <th>Status</th>
-            <th>Title</th>
-            {% for item in myitems %} 
-            <tr>
-                <td>{{ item.id }}</td>
-                <td>{{ item.status }}</td>
-                <td>{{ item.title }}</td>
-            </tr> 
-            {% endfor %}
-          </table>
+        {% for item in items %}
+          <li class="list-group-item flex-column">
+            <div class="d-flex w-100 justify-content-between">
+              <h5 class="mt-1 mb-1">{{ item.name }}</h5>
+              {% if item.idList == '5f5be2d2c3c4723ac911bbc9' %}
```

Comment:
It would be nicer to check the id and assign a status to the item in your python code.

We should try to keep the index.html sticking to a single responsibility - presenting the page, rather having to interpret the meaning of Trello's IDs.
Also, you now have these IDs repeated in your codebase (and far apart).